pom.xml

```diff
@@ -4,7 +4,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.saucelabs.bamboo</groupId>
     <artifactId>bamboo-sauceondemand-plugin</artifactId>
-    <version>1.6.78-SNAPSHOT</version>
```

Comment:
Why the change of version?

Snapshot is pretty java standard. Does the maven release tools handle other phrases?